authentication.go

```diff
@@ -6,16 +6,24 @@ import (
 	"net/http"
 	"time"
 
+	bip32 "github.com/bitcoin-sv/go-sdk/compat/bip32"
+	bec "github.com/bitcoin-sv/go-sdk/primitives/ec"
+	bscript "github.com/bitcoin-sv/go-sdk/script"
+	sighash "github.com/bitcoin-sv/go-sdk/transaction/sighash"
+
 	"github.com/bitcoin-sv/spv-wallet-go-client/utils"
 	"github.com/bitcoin-sv/spv-wallet/models"
 	"github.com/bitcoinschema/go-bitcoin/v2"
-	"github.com/libsv/go-bk/bec"
-	"github.com/libsv/go-bk/bip32"
 	"github.com/libsv/go-bt/v2"
-	"github.com/libsv/go-bt/v2/bscript"
-	"github.com/libsv/go-bt/v2/sighash"
 )
 
+// TODO: Issues with "github.com/bitcoinschema/go-bitcoin/v2"
+
+// TODO: Tx - go-sdk txJSON ?
+// TODO: NewTxFromString - no replacement in v2
```

Comment:
Isn't the 
`NewTransactionFromHex(str string) (*Transaction, error)` valid equivalent? Maybe we can adjust our usage?

Reply:
I mean there are few methods in package transaction of go-sdk.
There is transaction from bytes, from hex, from string - maybe we can play around and achieve the same?

Reply:
NewTransactionFromHex was used in place of NewTxFromString, I will look for the replacements of the other functions